jis-api/src/main/java/pl/edu/uj/JImageStream/api/core/Collector.java

```diff
@@ -1,8 +1,15 @@
 package pl.edu.uj.JImageStream.api.core;
 
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
 import java.awt.image.BufferedImage;
 
 
 public interface Collector<T> {
+    Logger logger = LogManager.getLogger("Collector");
     T collect(BufferedImage bufferedImage);
+    default void logMessage(){
+        logger.info(this.getClass().getSimpleName() + " collected image");
```

Comment:
it's way better to add simple logger.info(this.getClass() "successfully collected image"); in each collector in jis-plugins, we don't really need interface with log mechanism